mne/io/egi/tests/test_egi.py

```diff
@@ -27,40 +27,65 @@
 egi_mff_fname = op.join(egi_path, 'test_egi.mff')
 egi_mff_pns_fname = op.join(data_path(), 'EGI', 'test_egi_pns.mff')
 egi_pause_fname = op.join(egi_path, 'test_egi_multiepoch_paused.mff')
-egi_eprime_pause_fname = op.join(egi_path, 'test_egi_multiepoch_paused.mff')
+egi_eprime_pause_fname = op.join(egi_path, 'test_egi_multiepoch_eprime.mff')
 egi_pause_w1337_fname = op.join(egi_path, 'w1337_20191014_105416.mff')
 
+# absolute event times from NetStation
+egi_pause_events = {'AM40': [7.224, 11.928, 14.413, 16.848],
+                    'bgin': [6.121, 8.434, 13.369, 15.815, 18.094],
+                    'FIX+': [6.225, 10.929, 13.414, 15.849],
+                    'ITI+': [8.293, 12.997, 15.482, 17.918]}
+# absolute epoch times
+egi_pause_skips = [(1304000.0, 1772000.0), (8660000.0, 12296000.0)]
+
+egi_eprime_pause_events = {'AM40': [6.049, 8.434, 10.936, 13.321],
+                           'bgin': [4.902, 7.381, 9.901, 12.268, 14.619],
+                           'FIX+': [5.050, 7.435, 9.937, 12.322],
+                           'ITI+': [7.185, 9.503, 12.005, 14.391]}
+egi_eprime_pause_skips = [(1344000.0, 1804000.0)]
+
+egi_pause_w1337_events = None
+egi_pause_w1337_skips = [(21956000.0, 40444000.0), (60936000.0, 89332000.0)]
 
 @requires_testing_data
-@pytest.mark.parametrize('fname, n_pause, n_events, sfreq', [
-    (egi_pause_fname, 2, 18, 250),
-    (egi_eprime_pause_fname, 2, 18, 250),
-    (egi_pause_w1337_fname, 2, 0, 250),
+@pytest.mark.parametrize('fname, skip_times, event_times, sfreq', [
+    (egi_pause_fname, egi_pause_skips, egi_pause_events, 250),
+    (egi_eprime_pause_fname, egi_eprime_pause_skips, egi_eprime_pause_events, 250),
+    (egi_pause_w1337_fname, egi_pause_w1337_skips, egi_pause_w1337_events, 250),
 ])
-def test_egi_mff_pause(fname, n_pause, n_events, sfreq):
+def test_egi_mff_pause(fname, skip_times, event_times, sfreq):
     """Test EGI MFF with pauses."""
     with pytest.warns(RuntimeWarning, match='Acquisition skips detected'):
         raw = _test_raw_reader(read_raw_egi, input_fname=fname)
     assert raw.info['sfreq'] == sfreq
-    assert len(raw.annotations) == n_pause
-    if n_events == 0:
-        with pytest.raises(ValueError, match='Consider using .*events_from'):
-            find_events(raw)
```

Comment:
It looks like you have killed this useful part of the tests -- the `pytest.raises`. Can you put it back?

Also you need a rebase since I've pushed changes since you worked on this, unfortunately

Reply:
(If you don't know how to take care of this or get stuck working on it I can cherry-pick your commit and resolve the conflicts locally)